src/main/java/org/broadinstitute/hellbender/tools/FixCallSetSampleOrdering.java

```diff
@@ -28,13 +28,13 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
-@BetaFeature
+@ExperimentalFeature
 @CommandLineProgramProperties(summary = "Fix the sample names in a vcf that ran through the GenomicsDBImport when " +
         "the batch ordering bug was present, this will restore the correct sample names provided it is given the exact sample name mapping / vcf ordering" +
         "and batch sized that was used in the initial import. " +
         "See https://github.com/broadinstitute/gatk/issues/3682 for more information",
         oneLineSummary = "fix sample names in a shuffled callset",
-        programGroup = VariantProgramGroup.class,
+        programGroup = ShortVariantDiscoveryProgramGroup.class,
```

Comment:
Is this correct ? It really only matters if we expose it to doc/command line per above.

Reply:
My vote goes for Other program group.

Reply:
Done.